dask/bag/tests/test_bag.py

```diff
@@ -339,6 +339,36 @@ def test_map_partitions_with_kwargs():
         factor=2).sum().compute() == 2.0
 
 
+def test_random_sample_repeated_computation():
+    """
+    Repeated computation of a defined random sampling operation
+    generates identical results.
+    """
+    a = db.from_sequence(range(50), npartitions=5)
+    b = a.random_sample(0.2)
+    assert list(b) == list(b)  # computation happens here
```

Comment:
Should probably also test that the size of the result is in the expected range.  Perhaps use a larger sequence size and generous boundaries in order to avoid random failures.